mobile/src/main/res/values-fr/strings.xml

```diff
@@ -27,11 +29,14 @@
     <string name="settings_openhab_url">URL openHAB</string>
     <string name="settings_openhab_url_summary">Configurer manuellement l\'adresse openHAB. Cela désactive la découverte automatique si disponible. [%s]</string>
     <string name="settings_openhab_alturl">URL openHAB distante</string>
-    <string name="settings_openhab_alturl_summary">Utilisée lorsque l\'adresse local n\'est pas accessible [%s]</string>
+    <string name="settings_openhab_alturl_summary">Utilisée lorsque l\'adresse locale n\'est pas accessible [%s]</string>
     <string name="settings_openhab_username">Identifiant</string>
     <string name="settings_openhab_password">Mot de passe</string>
+    <string name="settings_openhab_sslclientcert">Certificat SSL client</string>
+    <string name="settings_openhab_sslclientcert_howto">Certificat SSL client - comment faire</string>
+    <string name="settings_openhab_sslclientcert_howto_summary">Taper ici pour avoir de l\'aide sur comment configurer l\'authentification par certificat SSL</string>
     <string name="settings_openhab_screentimeroff">Désactiver l\'extinction d\'écran</string>
-    <string name="settings_openhab_screentimeroff_summary">Désactive l\'extinction d\'écran lorsque l\'app openHAB fonctionne</string>
+    <string name="settings_openhab_screentimeroff_summary">Désactiver l\'extinction d\'écran lorsque l\'app openHAB fonctionne</string>
     <string name="settings_openhab_demomode">Mode démo</string>
     <string name="settings_openhab_demomode_summary">Lancer l\'app openHAB en mode démo</string>
     <string name="settings_openhab_theme">Thème app openHAB</string>
```

Comment:
Maybe you want to remove "app openHAB" from this string.

Reply:
Why ? "openHAB app" is now the official name in the store.

Reply:
I think it was changed to "Theme" in English and to "Design" in German.